samples/expert-finder-js/README.md

```diff
@@ -0,0 +1,92 @@
+
+## Interaction with app
+
+ ![expert-finder-gif](images/expert-finder.gif)
+
+## Try it yourself - experience the App in your Microsoft Teams client
+Please find below demo manifest which is deployed on Microsoft Azure and you can try it yourself by uploading the app package (.zip file link below) to your teams and/or as a personal app. (Sideloading must be enabled for your tenant, [see steps here](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading)).
+
+**Microsoft Teams Expert Finder sample app:** [Manifest](/samples/msgext-doc-compliance-checker/demo-manifest/expert-finder.zip)
+
+## Prerequisites
+
+- [Node.js 18.x](https://nodejs.org/download/release/v18.18.2/)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Blob Storage](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal)
+- [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
+
+## Run the app (Using Teams Toolkit for Visual Studio Code)
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio Code.
+
+1. Ensure you have downloaded and installed [Visual Studio Code](https://code.visualstudio.com/docs/setup/setup-overview)
+1. Install the [Teams Toolkit extension](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
+1. Select **File > Open Folder** in VS Code and choose this samples directory from the repo
+1. Using the extension, sign in with your Microsoft 365 account where you have permissions to upload custom apps
+1. Select **Debug > Start Debugging** or **F5** to run the app in a Teams web client.
+1. In the browser that launches, select the **Add** button to install the app to Teams.
+
+> If you do not have permission to upload custom apps (sideloading), Teams Toolkit will recommend creating and using a Microsoft 365 Developer Program account - a free program to get your own dev environment sandbox that includes Teams.
+
+## Setup and use the sample
+
+1) Create azure storage account
+   Refer the document [Create storage account]("https://learn.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal") and create a storage account in azure.
+
+2) Follow the steps mentioned to [Create azure table storage]("https://learn.microsoft.com/en-us/azure/storage/tables/table-storage-quickstart-portal") inside storage account.
+
+3) Follow schema as given in below image while creating azure table.
+   ![Schema](Images/table-storage-schema.png)
+
+4) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+5) Navigate to the `samples/expert-finder-js` folder and open with Visual Studio Code.
+
+6) Navigate to the `samples/expert-finder-js/.localConfigs` directory and update the values below.
+
+   ```txt
```

Comment:
Steps are missing to run this sample and upload the manifest.

Reply:
The test the app in copilot mentions the steps